### ISSUE TITLE
Shutdown timer and defaults

### ISSUE THREAD

**Issue by collectorgeneral** [budgie-desktop/budgie-desktop#993]
Currently running: Manjaro Linux V 17.0.2 with budgie-desktop 10.3.1

The current behaviour when the shutdown is triggered is a dialogue box asking for confirmation without a timer. I request that there be a timer that triggers shutdown after a certain number of seconds of inactivity, like the other DEs. Running certain applications such as [backintime](https://github.com/bit-team/backintime/) allows an option to shutdown when the task is done, but relies on having an automatic timer (see [this issue](https://github.com/bit-team/backintime/issues/788) I had reported).

**Comment by majamee** [comment]
This would be a highly welcomed feature. For reference I personally like Mate's DE implementation, even though I am not a Mate user:
It provides not only a timer, but also a perecentage bar which moves from 100% to 0% as the timer runs out:
Example given - ![](https://ubuntu-mate.community/uploads/default/original/2X/6/6b7db3aa8b190d6bcdad0e9629755c835ed6a5b2.png)

**Comment by irik77587** [comment]
would something like a [power-menu](https://github.com/irik77587/power-menu) work

**Comment by JoshStrobl** [comment]
> would something like a [power-menu](https://github.com/irik77587/power-menu) work?

No, that is a GNOME Shell extension. Budgie isn't related to Shell at all.

**Comment by irik77587** [comment]
I would only like a dialog box if I am about to Shutdown/Hibernate/Reboot. Otherwise no dialog

**Comment by buhtz** [comment]
Back In Time (BIT) maintainer here. Just for your info. BIT now implements its own shutdown countdown.
https://github.com/bit-team/backintime/pull/2102
